mrmustard/math/backend_manager.py

```diff
@@ -1100,23 +1149,28 @@ def sqrtm(self, tensor: Tensor, dtype=None) -> Tensor:
 
         Returns:
             The square root of ``x``"""
-        return self._apply("sqrtm", (tensor, dtype))
+        fn = self._get_fn("sqrtm")
+        return fn(tensor, dtype)
 
-    def sum(self, array: Tensor, axes: Sequence[int] = None):
+    def sum(self, array: Tensor, axis: Sequence[int] = None):
         r"""The sum of array.
 
         Args:
             array: The array to take the sum of
-            axes (tuple): axes to sum over
+            axis (tuple): The axis to sum over
```

Comment:
FYI originally it was "axes" because it's the plural of "axis"

Reply:
Right yeah my motivation for changing it is that both numpy and tensorflow use `axis` so I figured it's more intuitive to keep it the same as those